ABOUT.org

```diff
@@ -0,0 +1,285 @@
+parameterised and environment-aware configuration.
+
+Once reified, the system can be accessed via a selection of atoms in the ~kixi.datastore.application~ namespace, although this is mainly intended to
+facilitate the functions in the ~kixi.datastore.kaylee~ namespace.
+
+~kixi.datastore.web-server~ provides an HTTP server component which facilitates a limited number of queries (GET requests) into the system. All queries
+accept and respond using the ~transit+json~ format. There are routes which direct to 'unused' components, such as file segmentation. These should be
+avoided. The REST API in Datastore is in the process of being deprecated: For API access to the datastore, use ~witan.httpapi~. For querying facilities,
+use ~kixi.search~.
+
+*** FileStore
+
+| Key Namespaces                                 | Desciption                                                                 |
+|------------------------------------------------+----------------------------------------------------------------------------|
+| kixi.datastore.filestore.command-handler       | Handlers for incoming Commands, propagates Events                          |
+| kixi.datastore.filestore.commands              | Specs for Commands                                                         |
+| kixi.datastore.filestore.event-handler         | Handlers for incoming Events                                               |
+| kixi.datastore.filestore.events                | Specs for Events                                                           |
+| kixi.datastore.filestore.local                 | A FileStore implementation that stores files locally                       |
+| kixi.datastore.filestore.s3                    | A FileStore implementation that stores files in Amazon S3                  |
+| kixi.datastore.filestore.upload                | Specs for 'uploads'                                                        |
+| kixi.datastore.filestore.upload-cache.dynamodb | A FileStoreUploadCache implementation that records data in Amazon DynamoDB |
+| kixi.datastore.filestore.upload-cache.inmemory | A FileStoreUploadCache implementation that records data in an atom         |
+
+As its name suggests, the FileStore is the component responsible for handling /files/, specifically coordinating /uploads/ and /downloads/.
+In the case of an upload, it does not receive files, but manages uploads with S3 and provides addresses which clients use to upload there files directly.
+Similarly with downloads, an address is provided by S3 and clients are issued a redirect.
+
+The Commands and Events into this component surround 'initiating a file upload' and 'completing a file upload' which are, respectively, the start and
+end of the upload process.
+
+Downloads are provided via a GET ~/file/<id>/link~.
```

Comment:
which package does this route appear? Is this part of the REST API that should be deprecated in favour of kixi.search?